src/troute-network/troute/nhd_io.py

```diff
@@ -99,11 +98,43 @@ def read_config_file(custom_input_file):
     supernetwork_parameters = network_topology_parameters.get(
         "supernetwork_parameters", None
     )
+    # add attributes when HYfeature network is selected
+    if supernetwork_parameters['geo_file_path'][-4:] == "gpkg":
+        supernetwork_parameters["title_string"]       = "HY_Features Test"
+        supernetwork_parameters["geo_file_path"]      = supernetwork_parameters['geo_file_path']
+        supernetwork_parameters["flowpath_edge_list"] = None    
+        routelink_attr = {
+                        #link????
+                        "key": "id",
+                        "downstream": "toid",
+                        "dx": "length_m",
+                        "n": "n",  # TODO: rename to `manningn`
+                        "ncc": "nCC",  # TODO: rename to `mannningncc`
+                        "s0": "So",
+                        "bw": "BtmWdth",  # TODO: rename to `bottomwidth`
+                        #waterbody: "NHDWaterbodyComID",
+                        "tw": "TopWdth",  # TODO: rename to `topwidth`
+                        "twcc": "TopWdthCC",  # TODO: rename to `topwidthcc`
+                        "alt": "alt",
+                        "musk": "MusK",
+                        "musx": "MusX",
+                        "cs": "ChSlp"  # TODO: rename to `sideslope`
+                        }
+        supernetwork_parameters["columns"]             = routelink_attr 
+        supernetwork_parameters["waterbody_null_code"] = -9999
+        supernetwork_parameters["terminal_code"]       =  0
+        supernetwork_parameters["waterbody_null_code"] = -9999
```

Comment:
This line appears to be redundant (see L124).

Reply:
Thanks. I just removed it.